### PR TITLE
add information about how to make custom changes

### PR DESCRIPTION
**Pull request by renemennab**
Make it clear how one can add or alter specific rules to their repository without conflicting with the plugins rules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ npx install-peerdeps@2.0.1 --dev -o --yarn @intelie/eslint-config
 ```
 
 
+### If you need to make custom changes create a `.eslintrc` file in your project and add the new configuration there after extending the plugin:
```
Comment (melloc01) [inline README.md]: The extension can also be done on package.json, can you rephrase that to.

_If you need to make custom changes, extend this configuration:_

File: README.md [hunk 2]
```diff
@@ -24,6 +24,13 @@ npx install-peerdeps@2.0.1 --dev -o --yarn @intelie/eslint-config
...
+
+```json
+ "extends": "@intelie/eslint-config",
+ ..."your other rules"
```
Comment (melloc01) [inline README.md]: rules should be under the **rules** key and it's an array, right?

**Review by melloc01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (renemennab) [reply README.md]: rules is an object. i fixed it in the last commit

Comment (melloc01) [reply README.md]: cool, thanks
